app/controllers/user.js

```diff
@@ -1,33 +1,112 @@
 const User = require('../models').users,
-  logger = require('../logger');
+  errors = require('../errors'),
+  logger = require('../logger'),
+  jwt = require('jwt-simple'),
+  secret = process.env.NODE_API_JWT_SECRET_STRING,
+  { check, validationResult } = require('express-validator/check'),
+  bcrypt = require('bcrypt');
+
+
+exports.signup = (req, res, next) => {
```

Comment:
Are you still using views to tests your endpoints, I really suggest to get a little bit more confident with postman or some other tool like where you can make http requests.
Anyways if you want to still use this there is no problem, but remember to delete this once you tested your endpoint.

Reply:
This one popped up again after the rebase; I probably missed it when fixing the conflicts on each merge. My bad.